riff-raff/conf/routes

```diff
@@ -39,7 +39,7 @@ GET         /deployment/updates/:uuid                       controllers.DeployCo
 
 GET         /deployment/request/autoComplete/project        controllers.DeployController.autoCompleteProject(term:String)
 GET         /deployment/request/autoComplete/build          controllers.DeployController.autoCompleteBuild(project:String,term:String)
-GET         /deployment/request/history                     controllers.DeployController.deployHistory(project:String, stage:Option[String])
+GET         /deployment/request/history                     controllers.DeployController.deployHistory(project:String, stage:Option[String], isExactMatchProjectName:Option[Boolean])
```

Comment:
Should this new flag be exposed in the UI? The "navbar" that contains the controls could include this option as a checkbox?

Reply:
as discussed in person, that would require adding to the route for the 'history' page and wiring through to the `DeployFilter` plus the HTML etc to display such a tickbox - a worthwhile feature but something for another PR